.nuspec/Microsoft.Maui.Controls.MultiTargeting.targets

```diff
@@ -4,6 +4,12 @@
     <None Include="**\**\*.iOS.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <Compile Remove="**\iOS\**\*.cs" />
     <None Include="**\iOS\**\*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+  </ItemGroup>
+   <ItemGroup Condition=" '$(_MauiTargetPlatformIsMacCatalyst)' != 'True' ">
```

Comment:
I ended up not using this yet, but I think makes sense to add it no?

Reply:
No objections here to having the option :)